BioForm/BioForm/settings.py

```diff
@@ -49,6 +56,7 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
```

Comment:
You no longer need this middleware as you are now using a custom middleware for this purpose. 
Tip: You can even view Django code to see how things work.

Reply:
i did authentication using a backends custom authentication instead of a middleware one. Removing this caused this error. 
<img width="965" alt="image" src="https://user-images.githubusercontent.com/110596170/188613419-86e5a431-a577-4a34-a124-43febba65577.png">

I will try doing a custom authentication using middleware now and then remove this.